src/jats/jats-exporter.ts

```diff
@@ -906,7 +907,9 @@ export class JATSExporter {
       affiliations: () => '',
       contributors: () => '',
       table_element_footer: (node) =>
-        node.textContent ? ['table-wrap-foot', 0] : '',
+        node.childCount == 0
```

Comment:
Sometimes we receive an empty table wrap foot(no children at all), this is to fix it.